firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigLogger.java

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.remoteconfig.internal;
+
+import android.util.Log;
+import androidx.annotation.VisibleForTesting;
+
+/**
+ * A logger with settable log level. Defaults to level {@link Log#INFO}.
+ *
+ * @author danasilver
+ */
+public class ConfigLogger {
+  @VisibleForTesting static final String TAG = "FirebaseRemoteConfig";
+
+  private int logLevel;
+
+  public ConfigLogger() {
+    this.logLevel = Log.INFO;
```

Comment:
Would it makes sense to default to the highest level (`VERBOSE`) since the system level will override? (I see the [system level](https://developer.android.com/studio/debug/am-logcat#level) also defaults to the highest level.)

Reply:
Oh interesting, that's a good find. My intention was to mirror the system default, which I thought was `INFO` based on the [`isLoggable()`](https://developer.android.com/reference/android/util/Log#isLoggable(java.lang.String,%20int)) docs. I tested using `adb logcat` on the command line (rather than Android Studio, which has a dropdown to control the log level, so I'm not sure how its default works) and a verbose message appeared without setting any level.

Makes sense to use `VERBOSE` as the default for the RC logger, too.